assets/src/dashboard/app/views/shared/bodyViewOptions.js

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { Dropdown, ListBar } from '../../../components';
+import { STORY_SORT_MENU_ITEMS } from '../../../constants';
+
+const DisplayFormatContainer = styled.div`
+  margin: ${({ theme }) => `${theme.pageGutter.desktop}px 0`};
+  padding-bottom: 20px;
+  padding-left: 15px;
+
+  border-bottom: ${({ theme: t }) => t.subNavigationBar.border};
```

Comment:
Since we are no longer importing `theme` top level we do not have to rename this parameter.

Reply:
nope! good catch!